File: jscomp/test/gpr_2789_test.ml
let suites :  Mt.pair_suites ref  = ref []
let test_id = ref 0
let eq loc x y = Mt.eq_suites ~test_id ~suites loc x y 



let () = 
  eq __LOC__ 0 (Weak.length (Weak.create 0));
  eq __LOC__ 1 (Weak.length (Weak.create 1))

;; Mt.from_pair_suites __FILE__ !suites

File: jscomp/test/gpr_2789_test.js
'use strict';

var Mt = require("./mt.js");

var suites = {
  contents: /* [] */0
};

var test_id = {
  contents: 0
};

function eq(loc, x, y) {
  return Mt.eq_suites(test_id, suites, loc, x, y);
}

Mt.from_pair_suites("Gpr_2789_test", suites.contents);

exports.suites = suites;
exports.test_id = test_id;
exports.eq = eq;
/*  Not a pure module */
